api/handlers_test.go

```diff
@@ -18,7 +18,34 @@ import (
 )
 
 var (
-	mu sync.Mutex
+	mu         sync.Mutex
+	recipeTest = `{
+					"id" : "123", 
```

Comment:
I did this so that it is more easily readable and shows the structure more clearly. I can combine them all into one line. Is this what you are referring to?

Reply:
No that isn't what i mean. The indentation for each line can be reduced, so it looks like:

```
recipeTest = `{
		"id" : "123", 
		"alias" : "test", 
		"format" : "v4", 
		"files" : [
			{
				"description" : "test files"
			}
		], 
		"output_instances" : [
			{
				"dataset_id" : "1234",
				"editions" : [ 
					"edition-test"
				],
				"title" : "test",
				"code_lists" : [ 
					{
						"id" : "12345",
						"href" : "http://localhost:22400/code-lists/12345",
						"name" : "codelist-test-name",
						"is_hierarchy" : false
					}
				]
			}
		]
	}`
```

Reply:
Done in "Update different fields of recipe and add endpoint"